app/workers/edx_course_worker.rb

```diff
@@ -33,6 +33,7 @@ def handle_response_data(response_data)
     profed_track_type = CourseTrackType.find_by(type_of_achievement: 'edx_profed_certificate')
 
     response_data.each do |xml_doc|
+      language = xml_doc.xpath('//channel/language')
```

Comment:
Are you sure about that line? I would assume it should be `language = xml_doc.xpath('//channel/language').text`. Otherwise, the test will fail (as `<language>en</language>` is saved in the database)